app/assets/javascripts/backbone/views/page_view.coffee

```diff
@@ -0,0 +1,12 @@
+class MS.Views.PageView extends Backbone.View
+
+  el: 'body'
+
+  events: (params) ->
+    'change select[name="division_id"]': 'selectDivision'
```

Comment:
If el is just the form, then this becomes just `'change select': 'selectDivision'`